### ISSUE TITLE
Uncaught BufferedProcessError after installing ternjs

### ISSUE THREAD

**Issue by mslotema** [tststs/atom-ternjs#107]
I use Atom version 1.0.5 (it just got updated) and this problem arises when I install/enable atom-ternjs (currently on version 0.7.1). Disable it, and the error gows away. I've tried several combinations aof enabling and disabling extensions, but ternjs is the only one that causes trouble. So it seems.

```
Uncaught BufferedProcessError: Failed to spawn command `/Applications/Atom.app/Contents
/Frameworks/Atom Helper.app/Contents/MacOS/Atom Helper`. 
Make sure `/Applications/Atom.app/Contents/Frameworks/Atom Helper.app
/Contents/MacOS/Atom Helper` is installed and on your PATH
```
Also mentioned here [https://github.com/atom/atom/issues/7594]

**Comment by mrmckeb** [comment]
I've also faced this.

**Comment by diessica** [comment]
Same. It's happening to everyone.

**Comment by diessica** [comment]
Same here.

**Comment by gcrabtree** [comment]
Yep, running into it here on Atom 1.0.7and atom-ternjs 0.7.1

Running Atom Helper from the CL gives me:
`Error: Cannot find module '/Applications/Atom.app/Contents/Frameworks/Atom Helper.app/Contents/Resources/atom.asar/browser/lib/init.js'
    at Function.Module._resolveFilename (module.js:328:15)
    at Function.Module._load (module.js:270:25)
    at Function.Module.runMain (module.js:493:10)
    at startup (node.js:120:16)
    at node.js:877:3`

Running ./Atom --safe quiets down the bug.

**Comment by VorT3x** [comment]
Atom 1.0.11, atom-ternjs 0.7.2, got same problem:
```
Failed to spawn command /Applications/Atom.app/Contents/Frameworks/Atom Helper.app/Contents/MacOS/Atom Helper. Make sure /Applications/Atom.app/Contents/Frameworks/Atom Helper.app/Contents/MacOS/Atom Helper is installed and on your PATH
```

**Comment by dillmatic** [comment]
Same problem on Atom 1.0.19

**Comment by MethodGrab** [comment]
Atom 1.0.19 / atom-ternjs 0.7.3 / Node 4.1.1 / OS X 10.10.4

I was getting this error but closing all my open projects, quitting Atom and restarting it seemed to fix it!

![image](https://cloud.githubusercontent.com/assets/2421069/10189736/1db02932-6760-11e5-8105-612964734404.png)

**Comment by drawcard** [comment]
Having this problem as well. 

![screenshot_357](https://cloud.githubusercontent.com/assets/1435638/10297347/37a8fa36-6c19-11e5-98ea-446d1cef3240.jpg)

![screenshot_356](https://cloud.githubusercontent.com/assets/1435638/10297350/3afab85a-6c19-11e5-88dd-192af036ccf0.jpg)

**Comment by starquake** [comment]
I get this error when opening preferences with the ternjs plugin enabled.

**Comment by zmLGBBM** [comment]
Same problem here with Windows 10 and atom 1.0.19

**Comment by jordanbtucker** [comment]
Sorry, referenced the wrong PR. #144 fixes this (probably), #145 does not.
